lib/grandpa/message_tracker.go

```diff
@@ -93,6 +101,7 @@ func (t *tracker) handleBlock(b *types.Block) {
 	h := b.Header.Hash()
 	if vms, has := t.voteMessages[h]; has {
 		for _, v := range vms {
+			// handleMessage would never error for vote message
```

Comment:
maybe update the warn log? :p

Reply:
not sure what you want here? Add this comment in log or remove the log?